packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -18,14 +18,16 @@ import { ITheme, createTheme } from '../../Styling';
 
 const getClassNames = classNamesFunction<IFabricStyleProps, IFabricStyles>();
 const getFabricTheme = memoizeFunction((theme?: ITheme, isRTL?: boolean) => createTheme({ ...theme, rtl: isRTL }));
-const getDir = memoizeFunction((theme?: ITheme, dir?: IFabricProps['dir']) => {
-  if (dir) {
-    return dir;
-  }
-  if (theme && theme.rtl !== undefined) {
-    return theme.rtl ? 'rtl' : 'ltr';
-  }
-  return getRTL() ? 'rtl' : 'ltr';
+
+const getAllDir = memoizeFunction((theme?: ITheme, dir?: IFabricProps['dir']) => {
```

Comment:
`getAllDir`  - can you renaming this method to be more meaningful? or leave it as `getDir`?